CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.8.7
```

Comment:
Only one request, could you wait to cut the tag til #991 is in? Want to get that one in as soon as possible since it's caused breakages for some users

Reply:
of course!